composer.json

```diff
@@ -75,7 +75,7 @@
         "willdurand/hateoas-bundle":            "@stable",
         "winzou/state-machine":                 "~0.1",
         "winzou/state-machine-bundle":          "~0.2",
-        "payum/omnipay-bridge": "0.13.*"
+        "payum/omnipay-bridge":                 "~0.12"
```

Comment:
Also there should be `provide` in `payum/payum` package as there is `replace` note (for version `0.12` ofc).